src/main/resources/templates/ProductList.html

```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE HTML>
-<html xmlns:th="http://localhost" xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorator="default">
+<html xmlns:th="http://localhost" xmlns:layout="http://www.w3.org/1999/xhtml">
 <head>
     <title th:text="'Products ' + (${category} ? '(' + ${category} + ')' :'') + ' | 8-BLISS'"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link type="text/css" th:href="@{/css/bootstrap.min.css}" rel="stylesheet" />
+	<link type="text/css" th:href="@{/css/main.css}" rel="stylesheet" />
 </head>
 <body>
-    <div layout:fragment="content">
+    <div id="wrapper">
+        <th:block layout:replace="fragments/Nav"/>
+        <th:block layout:replace="fragments/Subnav"/>
```

Comment:
I get an exception relating to this when I attempt to view any of the `productList` pages

Reply:
`Subnav` is actually called `SubNav` :stuck_out_tongue: